documentation/modules/ROOT/partials/modules/all-connectors/ref-connector-monitoring-incremental-snapshot-metrics.adoc

```diff
@@ -0,0 +1,27 @@
+The connector also provides the following additional snapshot metrics when incremental snapshot is executed:
+
+[cols="45%a,25%a,30%a",options="header"]
+|===
+|Attributes |Type |Description
+
+|[[connectors-insap-metric-chunkid_{context}]]<<connectors-isnap-metric-chunkid_{context}, `ChunkId`>>
```

Comment:
There's an inconsistency between "insap" and "isnap". Also, what should this mean to begin with?

Reply:
@gunnarmorling It should be `insnap` - incremental snapshot